test/mongorito.test.js

```diff
@@ -134,14 +109,14 @@ describe ('Mongorito', function () {
       yield* Post.index('title');
 
       let indexes = yield* Post.indexes();
-      indexes.should.have.property('title_1');
+      indexes.should.have.deep.property('[' + (indexes.length - 1) + '].name', 'title_1');
```

Comment:
If you want to test array items, `deep.property` with a string argument is not a good way to do it. Just: `indexes[0].etc...`